dacite/types.py

```diff
@@ -63,6 +63,10 @@ def is_union(type_: Type) -> bool:
     return is_generic(type_) and type_.__origin__ == Union
 
 
+def is_tuple(type_: Type) -> bool:
```

Comment:
I know it's a very simple function, but we need some tests for it :)

Reply:
Do we really? `is_subclass` is more complex but [does not have tests](https://github.com/konradhalas/dacite/pull/70/files#diff-7ba398a2a24ad1b2ef39517206160a97c0907d26f8c0065a8fe3cb5da51efb93R125) (would be [here](https://github.com/konradhalas/dacite/blob/master/tests/test_types.py)). Tests for both could be added at the same time as a separate issue, or this function could be inlined to directly use `is_subclass` (may be clearer, anyways, and avoids a possible proliferation of `is_<typename>` helpers).

Reply:
Hi, I've added missing tests in [5544935](https://github.com/konradhalas/dacite/pull/175/commits/5544935e38bc3a615808e9083e26a07239f83335)

Reply:
hi @michalchrobok - I think (but I can be wrong) that you force-pushed only 1 commit (with tests). I don't see your previous changes.

Reply:
Same, @michalchrobok are you able to restore the original code you wrote with this test of it at the end?

Reply:
@konradhalas you were correct, I mistakenly force-pushed only tests, my bad. @mcgibbon I've restored original code, it should be possible to merge right now.